airflow-core/docs/core-concepts/executor/index.rst

```diff
@@ -239,7 +239,8 @@ Important BaseExecutor Methods
 These methods don't require overriding to implement your own executor, but are useful to be aware of:
 
 * ``heartbeat``: The Airflow scheduler Job loop will periodically call heartbeat on the executor. This is one of the main points of interaction between the Airflow scheduler and the executor. This method updates some metrics, triggers newly queued tasks to execute and updates state of running/completed tasks.
-* ``queue_command``: The Airflow Executor will call this method of the BaseExecutor to provide tasks to be run by the executor. The BaseExecutor simply adds the TaskInstances to an internal list of queued tasks within the executor.
+* ``queue_command``: Airflow 2 way of doing things. The Airflow Executor will call this method of the BaseExecutor to provide tasks to be run by the executor. The BaseExecutor simply adds the TaskInstances to an internal list of queued tasks within the executor. CeleryK8s and LocalK8s executors are examples of this.
```

Comment:
> CeleryK8s and LocalK8s executors are examples of this.

Those hardcoded hybrid executors don't have anything to do with the queue_command interface. _Any_ executor working with v2 core airflow will use this method.

Reply:
Yep that has been removed / reworked